### DISCUSSION TITLE
My Feedback on Zed

### DISCUSSION THREAD

**Discussion by hrasekj** [zed-industries/zed#15152] [Feature Requests]
Hello, I installed Zed and using it for a while. For OS i'm using Debian and Hyprland (which is using wayland). 

First of all, i love it. Just making some small tweaks but so far as React, Node.js dev worked everything as expected. I love the speed and autocomplete. I'm using Sublime Text for my daily work but Zed has much more in it :heart: 

I'm currious about extensions, how would they work. What are plans for them, in what language they have to be written?

There are some small issues I like to address here. That would make my experience with Zed even better if fixed/added.

1. Hover opens definitions twice
![image](https://github.com/user-attachments/assets/93d8a26d-6407-4f6e-b239-7d68955ec1be)

2. Cycling tabs should have option to switch between recently used tabs

3. When I want to open files via zed CLI i expect Zed to emit some sort of system event, that file was opened. For instance Firefox emits `urgent` event. I'm not sure what kind of event is this. But it helps to set focus to that window when I'm on different Workspace (in terms of system workspaces not Zed's).

4. When starting Zed my screen turns off for a second I guess it's someting with that GPU rendering support. Is it possible to tweak this or is it something i have to deal with?

5. Code actions menu should have some fuzzy find
6. Would be nice that Linter errors be more verbose, not only error message, but also come context, like what rule trigger the error and link to docs.

Thank you for your great work on Zed

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
